api/src/main/java/org/opennms/integration/api/v1/config/poller/Downtime.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.integration.api.v1.config.poller;
+
+import java.util.Optional;
+
+/**
+ * Downtime model. This determines the rates at which
+ * addresses are to be polled when they remain down for extended periods.
+ * Usually polling is done at lower rates when a node is down until a
+ * certain amount of downtime at which the node is marked
+ * 'deleted'.
+ */
+public interface Downtime{
+
+    /**
+     * Determines if service is to be deleted when down
+     * continuously since the start time.  Use 'never' to never delete
+     * services, 'managed' to only delete the service if it is on an
+     * interface that is managed, or 'always' to delete even if the
+     * service is on an unmanaged interface.
+     */
+    enum DeletingMode {
+        ALWAYS,
+        MANAGED,
+        NEVER
+    }
+
+    /**
+     * Start of the interval.
+     */
+    long getBegin();
```

Comment:
Can we make this of type `Duration`? Not sure if this will work with serialization...